docs/content/tutorials/custom-listen-ports.md

```diff
@@ -0,0 +1,158 @@
+
+```nginx
+server {
+    listen 85{{ if $s.ProxyProtocol }} proxy_protocol{{ end }};
+
+    server_name {{ $s.ServerName }};
+
+    set $resource_type "virtualserver";
+    set $resource_name "{{$s.VSName}}";
+    set $resource_namespace "{{$s.VSNamespace}}";
+```
+
+Modify the file you need (per the example above). In the example, we modified `nginx-plus-virtualserver.tmpl`:
+
+
+## Rebuild the NGINX Ingress Controller image
+
+You must rebuild the NGINX Ingress Controller image for the new port settings to take effect.
+Once the image is built and pushed, make sure you update your deployment to point to the new image and deploy.
+Once deployed, create a new `virtualServer` resource and run `nginx -T` to confirm if the port change has taken effect.
+
+Ensure that your `deployment` and your `service` match up to the new port you configured in the templates.
+Below is an example of  `deployment` and `service` matching to the new port that NGINX Ingress Controller now listens on.
+
+```nginx
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: nginx-ingress
+  namespace: nginx-ingress
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: nginx-ingress
+  template:
+    metadata:
+      labels:
+        app: nginx-ingress
+      annotations:
+       prometheus.io/scrape: "true"
+       prometheus.io/port: "9113"
+       prometheus.io/scheme: http
+    spec:
+      serviceAccountName: nginx-ingress
+      containers:
+      - image: nginx/nginx-ingress:3.0.2
+        imagePullPolicy: IfNotPresent
+        name: nginx-ingress
+        ports:
+        - name: http
+          containerPort: 85
+        - name: https
+          containerPort: 443
+        - name: readiness-port
+          containerPort: 8081
+        - name: prometheus
+          containerPort: 9113
+        readinessProbe:
+          httpGet:
+            path: /nginx-ready
+            port: readiness-port
+          periodSeconds: 1
+        securityContext:
+```
+
+Notice that now, the `http` port is set to `85`, which reflects the change we made in the template file.
+
+Here is the `service` file:
+
+```nginx
+apiVersion: v1
+kind: Service
+metadata:
+  name: nginx-ingress
+  namespace: nginx-ingress
+spec:
+  externalTrafficPolicy: Local
+  type: LoadBalancer
+  ports:
+  - port: 80
+    targetPort: 85
+    protocol: TCP
+    name: http
+  - port: 8443
+    targetPort: 8443
+    protocol: TCP
+    name: https
+  selector:
+    app: nginx-ingress
+```
+
+Since NGINX Ingress Controller is now listening on ports 85 and 8443, you must modify the `targetPort` in the NGINX Ingress Controller service to match the change in the deployment to ensure traffic will be sent to the proper port.
+
+The parameter to change above is `targetPort`. Since we have changed NGINX Ingress to listen on port 85, we need to match that in the service: requests will be sent to NGINX Ingress Controller on port 85 instead of the default value, port 80.
```

Comment:
```suggestion
The parameter to change above is `targetPort`. Since we have changed NGINX Ingress Controller to listen on port 85, we need to match that in the service: requests will be sent to NGINX Ingress Controller on port 85 instead of the default value, port 80.
```